gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/components/PickerPopup.kt

```diff
@@ -141,12 +141,14 @@ internal data class PickerPopupPositionProvider(
             IntOffset(offset.x.toPx().toInt(), offset.y.toPx().toInt())
         }
 
+        val displayPadding = with(density) { 16.dp.toPx().toInt() }
```

Comment:
❓ What do you think about passing the padding as a parameter?

Reply:
Done. I kept the default value for simplicity.